client/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import StyledIcon from "./IconStyles"
+
+export default function Icon({ color, children }: any) {
+  return <StyledIcon background={color}>{children}</StyledIcon>;
```

Comment:
Do we have to use `children`? That seems confusing to me, maybe use `name` or `iconName`?